datasets/uy/pep/crawler.py

```diff
@@ -0,0 +1,55 @@
+from zavod import Context
+from typing import Dict, List, Optional
+from pantomime.types import XLSX
+import openpyxl
+from normality import slugify
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+
+def sheet_to_dicts(sheet):
+    headers: Optional[List[str]] = None
+    for row in sheet.rows:
+        cells = [c.value for c in row]
+
+        if headers is None and all(cells):
+            headers = [slugify(h) for h in cells]
+            continue
+
+        if headers:
+            row = dict(zip(headers, cells))
+            yield row
+
+
+def parse_sheet_row(context: Context, row: Dict[str, str]):
+    person = context.make("Person")
+
+    id_number = row.pop("c-i")
+    person_name = row.pop("nombre")
+    role = row.pop("cargo")
+
+    person.id = context.make_slug(id_number, prefix="ur-cedula")
```

Comment:
```suggestion
    person.id = context.make_slug(id_number, prefix="uy-cedula")
```

Let's base the prefix on the [country code](https://www.opensanctions.org/reference/#type.country)

Reply:
Thanks for catcching that! I've changed the `ur` prefix to the correct country code, `uy` in the new commit